.github/workflows/publish-demo.yml

```
@@ -36,30 +36,20 @@ jobs:
           git-lfs: 'true'
       - name: 🚚 Pull Git LFS files
         run: git lfs pull
-
-      #  Setup Expo CLI action
-      # - name: Setup Expo
-      #   uses: expo/expo-github-action@v5
-      #   with:
-      #     expo-version: 3.x
-      #     expo-username: ${{ secrets.EXPO_CLI_USERNAME }}
-      #     expo-password: ${{ secrets.EXPO_CLI_PASSWORD }}
-      #     expo-cache: true
-
+      - name: Setup Expo
+        uses: expo/expo-github-action@v7
+        with:
+          expo-version: latest
+          eas-version: latest
+          token: ${{ secrets.COMMUNITY_EXPO_ACCESS_TOKEN }}
       - name: 🧶 Install workspace node modules
         if: steps.expo-caches.outputs.yarn-workspace-hit != 'true'
         run: yarn install --frozen-lockfile
       - name: 🔧 Install Expo CLI v3
         run: yarn global add expo-cli@3
-
-      # - name: Publish Expo app
-      #   working-directory: ./apps/native-component-list
-      #   run: expo publish --release-channel=pr-${{ github.event.number }}
-      # # Get Expo link for the comment
-      # - name: Get expo link
-      #   id: expo
-      #   run: echo "::set-output name=path::@community/native-component-list?release-channel=pr-${{ github.event.number }}"
-
+      - name: Publish Expo app
+        working-directory: ./apps/native-component-list
+        run: EXPO_SDK_VERSION=43.0.0 eas update --branch main --message "initial publish"
```

I think we may want to use a branch other than `main` so that this doesn't send updates to production builds of NCL if we were to reintroduce those, at least for now. And the message needs to be changed, maybe something like this?
```suggestion
        run: EXPO_SDK_VERSION=43.0.0 eas update --branch test --message "Publishing $GITHUB_SHA"
```

---

We'd choose whatever channel/branch we wanted for the production build if/when we introduce them. The way I think of this is we're publishing to the `main` branch (which I still think of as `master`). If there were a production build, it would have channel "production", and we'd point it to the current release branch. We'd then have another build, "rc", which would point to the next release branch. We'd have a third build, "development", which can point to the `main` branch. Setting this up is a larger task though.